package.json

```diff
@@ -39,10 +38,21 @@
     "demo": "./cli.js 'test/fixtures/**/*'",
     "lint": "tslint --project .",
     "lint.fix": "tslint --fix --project . ",
-    "test": "mocha --compilers ts:ts-node/register 'test/**/*Tests.ts'",
+    "test": "jest",
     "tsc": "tsc --project . && echo 'OK'",
     "watch": "tsc --project . --watch"
   },
+  "jest": {
```

Comment:
IMO this clutters package.json too much, please move this to its own jest.config.json file